src/app/index.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from 'react';
+import { hot } from 'react-hot-loader';
+import './app.scss';
+import * as mac from 'assets/imgs/mac.png';
+import * as windows from 'assets/imgs/windows.png';
+import * as linux from 'assets/imgs/linux.png';
+import * as Secure from 'assets/imgs/icon-shield.svg';
+import * as Offline from 'assets/imgs/icon-offline.svg';
+import * as Yours from 'assets/imgs/icon-hdd.svg';
+import * as OpenSource from 'assets/imgs/icon-github.svg';
```

Comment:
Why are these `* as`, and why are they untyped?

Reply:
With typescript it's the only way I can import them without errors. Not exactly sure why.